File: server/aoNewsletter2/Models/Db/NewsletterAdBannerLayoutModel.vb
Imports System
Imports Contensive.Addons.Newsletter.Models.Db
Imports Contensive.BaseClasses
Imports Contensive.Models.Db

Namespace Models.Db
    Public Class NewsletterAdBannerLayoutModel
        Inherits Global.Contensive.Models.Db.DbBaseModel

        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Newsletter Ad Banner Layouts", "NewsletterAdBannerLayouts", "default", False)
        '
        Public Property rowcnt As Integer
        Public Property columncnt As Integer
        Public Property pxcolumnspace As Integer
        Public Property pxrowspace As Integer
        '
    End Class
End Namespace

File: server/aoNewsletter2/Models/Db/NewsletterAdBannerLayoutModel.cs
using Contensive.Models.Db;

namespace Contensive.Addons.Newsletter.Models.Db {
    public class NewsletterAdBannerLayoutModel : DbBaseModel {

        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Newsletter Ad Banner Layouts", "NewsletterAdBannerLayouts", "default", false);
        // 
        public int rowcnt { get; set; }
        public int columncnt { get; set; }
        public int pxcolumnspace { get; set; }
        public int pxrowspace { get; set; }
        // 
    }
}